Rakefile

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
```

Comment:
Can we just manually manage the changelog and not include any of this? It should just be:
```
require 'puppetlabs_spec_helper/rake_tasks'
require 'puppet_blacksmith/rake_tasks' if Bundler.rubygems.find_name('puppet-blacksmith').any?
```